src/types/DateTimeType.ts

```diff
@@ -0,0 +1,3 @@
+import { DATE, TIME } from '../core/data/constants';
+
+export type ModeType = typeof DATE | typeof TIME;
```

Comment:
Il me semble que le type défini ici revient à `type ModeType = string | string;`

Reply:
hum je ne crois pas 
tu peux faire le test suivant : 
dans le fichier `EventDateTime`, tu peux mettre `<TouchableOpacity style={styles.dateCell} onPress={() => onPress('test')} disabled={disabled}>` (changer le DATE en 'test' dans le onPress) et du coup tu as une erreur ts `Argument of type '"test"' is not assignable to parameter of type 'ModeType'.ts(2345)`